src/sdk/namespace_cache.js

```diff
@@ -739,6 +741,35 @@ class NamespaceCache {
         return this.namespace_hub.get_blob_block_lists(params, object_sdk);
     }
 
+    //////////
+    // ACLs //
+    //////////
+
+    async get_object_acl(params, object_sdk) {
+        if (config.NAMESPACE_CACHING.ACL_HANDLING === "reject") {
+            throw new S3Error(S3Error.AccessDenied);
+        }
+
+        if (config.NAMESPACE_CACHING.ACL_HANDLING === "pass-through") {
+            return this.namespace_hub.get_object_acl(params, object_sdk);
+        }
+
+        return {
+            owner: s3_utils.DEFAULT_S3_USER,
+            access_control_list: [{
+                Grantee: { ID: s3_utils.DEFAULT_S3_USER, Type: "CanonicalUser" },
+                Permission: "FULL_CONTROL"
+            }]
+        };
```

Comment:
I think before we return the default ACL we need to check if the object exists, just like we do in namespace_nb, but in this case our call to this.read_object_md() will use the namespace_cache implementation:
```
            await this.read_object_md({ ... }, object_sdk);
            return s3_utils.DEFAULT_OBJECT_ACL; // see other comment on moving this to s3_utils...
```